contracts/Strategy.sol

```diff
@@ -229,9 +231,12 @@ contract Strategy is BaseStrategy {
 
             // Now we can withdraw all
             fMint.mustWithdraw(address(want), balanceOfCollateral());
-        } else {
+        } else if (_targetDebt == balanceOfDebt()) {
```

Comment:
I don't understand why this was added.

Reply:
this is the case in which the reduceDebt function has successfully reached targetDebt

I believe we should contemplate the scenario in which that is not the case. What happens if the targetDebt is still higher than current debt? 

last case is the same than the first (targetDebt == 0 & balanceOfDebt > 0, meaning that debt has not been paid in full, even if it was the intention) but with targetDebt != 0

maybe we can put those two cases together as 

`targetDebt != balanceOfDebt()`